graphql-mem-prof.md

```diff
@@ -0,0 +1,57 @@
+# graphql memory profiling with heaptrack and disabled auth
```

Comment:
is this not memory profiling for everything ... why specific instructions for graphql ?

Reply:
> is this not memory profiling for everything

that's correct :+1: 

> why specific instructions for graphql ?

This test was to help with this investigation only https://github.com/trustification/trustify/issues/750

Reply:
I start to agree that we are not required to land the PR.. 
I'm thinking in a way to have a handy graphql-specific-mem-prof-only. 
:thinking: even if we close the PR the register will continue here so we can replicate